crates/protogen/src/sqlexec/physical_plan.rs

```diff
@@ -270,8 +270,8 @@ pub struct DeleteExec {
 
 #[derive(Clone, PartialEq, Message)]
 pub struct InsertExec {
-    #[prost(message, tag = "1")]
-    pub table: Option<TableEntry>,
+    #[prost(bytes, tag = "1")]
+    pub provider_id: Vec<u8>, // UUID
```

Comment:
I see we do this in other places, but I read this and thought "surely there's a way to avoid needing to specify the type in a comment, and it looks like https://crates.io/crates/prost-uuid/0.2.0 is a thing that exists? 

Need not change it everywhere here or even do it in this PR, but does this seem unreasonable?

Reply:
Yeah, I think we can do it. Will start from this PR.

Reply:
So, the issue here is the `uuid` version used by this library. We can implement this on our own and use it. I'll create an issue to track it.

Reply:
1884